gdb.spec

```diff
@@ -3,7 +3,7 @@ Source: https://ftp.gnu.org/gnu/%{n}/%{n}-%{realversion}.tar.gz
 
 Patch0: gdb-disable-makeinfo
 
-Requires: python zlib xz expat
+Requires: python3 zlib xz expat
```

Comment:
@davidlange6 , I think we can not change it unless cmsRun start using python3. 
For PY3 IBs, I had to build gdb with python3 ( to match cmsRun python) otherwise it fails/hang when cmsRun starts it ( to get the stacktrace)

Reply:
ah - didn't realize we used the python interface.  then I should revert this one